.github/workflows/qat.yml

```diff
@@ -16,74 +16,64 @@ on:
-          nodejs: 1
-
-      - name: Check QAT Script and Feature Branch
-        run: |
-          has_qat=`jq -r '.scripts|to_entries[]|select(.key=="${{ env.NPM_QAT_SCRIPT }}")|.key' package.json 2>/dev/null | head -n 1 | wc -l`
-          if [ $has_qat -eq 1 ] && [[ "${{ github.head_ref || github.ref_name }}" =~ 'feature/' ]]; then
-            echo "RUN_QAT=true" >> $GITHUB_ENV
-          else
-            echo "Not a feature branch or qat script not found. Skipping Playwright tests."
-            echo "RUN_QAT=false" >> $GITHUB_ENV
-          fi
-
-      - name: Check PMCQA
-        if: ${{ env.RUN_QAT == 'true' && inputs.THEME_NAME  != '' && inputs.QA_SITE_DOMAIN != '' }}
-        timeout-minutes: 10
-        run: |
-          # `version-hash` is created by jenkins script when a branch is pushed
-          # https://github.com/penske-media-corp/pmc-jenkins-scripts/blob/8ee0244a8485b61cfa6d3f9af00f7e6a55ddb042/bin/push-pmcqa-repo.sh#L36-L38
-          hash_url="https://${{ env.FEATURE_BRANCH_NAME }}.${{ inputs.QA_SITE_DOMAIN }}/wp-content-vipgo-sites/${{ env.FEATURE_BRANCH_NAME }}/themes/vip/${{ inputs.THEME_NAME }}/version-hash"
-          echo "HASH_URL: $hash_url"
-          hash=`curl --no-progress-meter --insecure $hash_url`
-          until [[ $hash == ${{ github.event.pull_request.head.sha }} ]]; do
-              echo -n "Waiting for Jenkins Deployment ... \n"
-              hash=`curl --no-progress-meter --insecure $hash_url`
-              sleep 5
-          done
-
-      - name: Install Playwright
-        if: ${{ env.RUN_QAT == 'true' }}
-        run: |
-          . "$NVM_DIR/nvm.sh"
-
-          nvm install
-
-          # Playwright needs to be added in package.json as dependency
-          npm ci
-
-      - name: Run Playwright tests
-        if: ${{ env.RUN_QAT == 'true' }}
-        run: |
-          if [ "${{ inputs.QA_SITE_DOMAIN }}" != '' ]; then
-            export ENVIRONMENT_URL="https://${{ env.FEATURE_BRANCH_NAME }}.${{ inputs.QA_SITE_DOMAIN }}"
-          fi
-          npm run ${{ env.NPM_QAT_SCRIPT }}
+          fetch-depth: 0 # Check out the full git history
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16 # Checkly CLI requires Node.js 16 or higher.
+      - name: Restore or cache node_modules # Restore node_modules cache if available
+        id: cache-node-modules
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('package-lock.json') }}
+      - name: Install dependencies # Install NPM dependencies if cache miss
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: npm ci
+      - name: Run checks # Run Checkly checks and record test session
+        id: run-checks
+        run: npx checkly test -e ENVIRONMENT_URL=${{ env.ENVIRONMENT_URL }} -e CHECKLY_REPO_SHA=${{ env.CHECKLY_REPO_SHA }} --reporter=github --record
```

Comment:
Re `ENVIRONMENT_URL` previously, with the old step (being deleted here) `name: Check PMCQA` we checked for the per-branch QA site to be created (i.e. so we wait to run tests until jenkins finishes building the QA site)

Do we no longer need to do that?

Reply:
In the SJ PR I see `QA_SITE_DOMAIN: sourcingjournal.pmcqa.com` which maps to `ENVIRONMENT_URL: ${{ inputs.QA_SITE_DOMAIN }}` above. 

Does this mean that the QAT will run against the main qa site? 

We purposefully want the QAT to run on the per-branch site so that, for example, a PR which breaks the Login button is blocked from being merged. Or, if you add a new element to the site with an accompanying QAT test, that the test can verify the new element in the PR.